parsons/zoom/zoom.py

```diff
@@ -275,3 +275,103 @@ def get_webinar_registrants(self, webinar_id):
         tbl = self._get_request(f"webinars/{webinar_id}/registrants", "registrants")
         logger.info(f"Retrieved {tbl.num_rows} webinar registrants.")
         return tbl
+
+    def get_meeting_poll(self, meeting_id: str, poll_id: str) -> Table:
```

Comment:
The typing for the meeting ID's is off in the connector. Webinars do have string UUIDs, but meetings have integer IDs. Normally I'm in favor of adding type annotations, but not if it will conflict with other areas of the code. Maybe you could just remove the "str" type annotation for the meeting_ids in the code, and change the type to int in the docstrings? Then we could clean the rest of the file later?